include/swift/Driver/DependencyGraph.h

```diff
@@ -68,12 +70,13 @@ class DependencyGraphImpl {
   class MarkTracerImpl {
     class Entry;
     llvm::DenseMap<const void *, SmallVector<Entry, 4>> Table;
+    UnifiedStatsReporter *Stats;
 
     friend class DependencyGraphImpl;
   protected:
-    MarkTracerImpl();
+    MarkTracerImpl(UnifiedStatsReporter *Stats);
```

Comment:
Haha I _did_ fix it in one place (the subclass). Just not _two_ places.

Brains. Sorry! Pushed ever-more-fixy fix.